proposed/dotnet-bootstrapper/dotnet-bootstrapper-path.md

```diff
@@ -0,0 +1,130 @@
+
+### Configure Visual Studio to set DOTNET_ROOT
+
+We could modify Visual Studio to check if `DOTNET_HOME` is in the `PATH`, and that the `paths` option in `global.json` is not set, then have Visual Studio set its process tree's `DOTNET_ROOT` to be the admin location. This would preserve Visual Studio's behavior while allowing it to use the appropriate SDK context. Care would need to be taken to not override any custom DOTNET_ROOT setting.
+
+#### Advantages and Disadvantages
+
+While this approach addresses some security concerns related to the admin/local boundary, it creates a situation where local installs performed via the CLI would not work with Visual Studio, potentially causing user confusion. This also may cause our tooling to break other IDEs and require action from JetBrains and others, which is a large downside and why this is at the bottom of the list.
+
+## Technical Details
+
+If `bootstrapper` sets the user path to enable a folder of local .NET SDKs to be discovered by the `dotnet.exe` application host at `DOTNET_ROOT`, via the `PATH`, those installs would become undetectable to the runtime, debugger, etc. when Visual Studio updates and modifies the System `PATH`. With changes in .NET 7+, `DOTNET_MULTI_LEVEL_LOOKUP` is no longer enabled, and once an install is detected in one location, the lookup does not fall back. Thus, the break to `bootstrapper` occurs because when Visual Studio automatically updates, it modifies the `PATH`, which can cause the `bootstrapper`'s managed installs to no longer be found.
+
+On the other hand, if `bootstrapper` set the system `PATH`, it would modify the behavior of Visual Studio, as Visual Studio would no longer find its installs. This could be considered a breaking change to Visual Studio. In this scenario, Visual Studio may begin to run on a different SDK than the one it shipped with, based on whichever SDKs the user has installed that can be used with the user project (based on the major.minor version of .NET and host resolution logic for the specific context.) This is already possible today by installing a different administrative SDK that is newer than the one Visual Studio has. However, it is not possible to install a downgrade of an administrative install in the specific version-set without uninstalling the SDK owned by Visual Studio first. This also means that updates of the SDK that Visual Studio includes would no longer take effect within Visual Studio, as the update mechanism would need to be handled by `bootstrapper`.
+
+An additional concern would be the `source-built` SDKs and the `pkg` installer.
+
+There have been several methods suggested to handle this, which are listed below.
```

Comment:
I think you can remove this line, I think it made sense when this section was at the beginning of the document, but not now.